pkg/controller/manager/manager_controller.go

```diff
@@ -458,14 +458,19 @@ func (r *ReconcileManager) Reconcile(ctx context.Context, request reconcile.Requ
 		return reconcile.Result{}, err
 	}
 
-	clusterConfig, err := utils.GetElasticsearchClusterConfig(context.Background(), r.client)
-	if err != nil {
-		if errors.IsNotFound(err) {
-			r.status.SetDegraded(operatorv1.ResourceNotFound, "Elasticsearch cluster configuration is not available, waiting for it to become available", err, logc)
-			return reconcile.Result{}, nil
+	var clusterConfig *relasticsearch.ClusterConfig
+	// We only require Elastic cluster configuration when Kibana is enabled. We infer whether Kibana is enabled by checking
+	// FIPS configuration mode and multi-tenancy mode. See manager.go function kibanaEnabled for more details.
```

Comment:
We should probably make `kibnanEnabled` public and use the same implementation everywhere that we need to check if Kibana is enabled, so that they don't get out of sync.